GarupaPico/GarupaPico/TabbedEventTopPage.xaml.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace GarupaPico
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class TabbedEventTopPage : TabbedPage
```

Comment:
The content of a `TabbedPage` should be `ContentPage` unless you are having sub level tabs.